### PR TITLE
Adapt to Ansible 2.10

### LINKED ISSUE #8: Inclusion of skydive.skydive in Ansible 2.10
This collection will be included in Ansible 2.10 because it contains modules and/or plugins that were included in Ansible 2.9. Please review:

* The list of requirements for inclusion in Ansible: https://github.com/ansible-collections/overview/blob/master/collection_requirements.rst
* The roadmap with all important dates for Ansible 2.10: https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst

## DEADLINE: 2020-08-18

The latest version of the collection available on August 18 will be included in Ansible 2.10.0, except possibly newer versions which differ only in the patch level. (For details, see [the roadmap](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst)). Please release version 1.0.0 of your collection by this date! **If 1.0.0 does not exist, the same 0.x.y version will be used in all of Ansible 2.10 without updates,** and your 1.x.y release will not be included until Ansible 2.11 (unless you request an exception at a [community working group meeting](https://github.com/ansible/community/issues/539) and go through a demanding manual process to vouch for backwards compatibility . . . you want to avoid this!).

## Follow semantic versioning rules

Your collection versioning must follow all [semver rules](https://semver.org/). This means:

* Patch level releases can only contain bugfixes;
* Minor releases can contain new features, new modules and plugins, and bugfixes, but must not break backwards compatibility;
* Major releases can break backwards compatibility.

## Changelogs and Porting Guide

Your collection should provide data for the Ansible 2.10 changelog and porting guide. The changelog and porting guide are automatically generated from ansible-base, and from the changelogs of the included collections. All changes from the `breaking_changes`, `major_changes`, `removed_features` and `deprecated_features` sections will appear in both the changelog and the porting guide. You have two options for providing changelog fragments to include:

1. If possible, use the [antsibull-changelog tool](https://github.com/ansible-community/antsibull-changelog/), which uses the same changelog fragment as the ansible/ansible repository (see the [documentation](https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst)).
2. If you cannot use antsibull-changelog, you can provide the changelog in a machine-readable format as `changelogs/changelog.yaml` inside your collection (see the [documentation of changelogs/changelog.yaml format](https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelog.yaml-format.md)).

If you cannot contribute to the integrated Ansible changelog using one of these methods, please provide a link to your collection's changelog by creating an issue in https://github.com/ansible-community/ansible-build-data/. If you do not provide `changelogs/changelog.yml` or a link, users will not be able to find out what changed in your collection from the Ansible changelog and porting guide.

## Make sure your collection passes the sanity tests

Run `ansible-test sanity --docker -v` in the collection with the latest [ansible-base](https://pypi.org/project/ansible-base/) or `stable-2.10` ansible/ansible checkout.

## Keep informed

Be sure you're subscribed to:

* [Changes impacting Collections](https://github.com/ansible-collections/overview/issues/45) to track changes that Collection maintainers should be aware of;
* The Bullhorn, a newsletter for the Ansible developer community, [back issues and how to add content](https://github.com/ansible/community/issues/546).

## Questions and Feedback

If you have questions or want to provide feedback, please see [the Feedback section in the collection requirements](https://github.com/ansible-collections/overview/blob/master/collection_requirements.rst#feedback).

(Internal link to keep track of issues: ansible-collections/overview#102)

### PR DESCRIPTION
**Pull request by adrianlzt**
##### SUMMARY
Adapt repo to requirements to be added in Ansible 2.10

Fixes #8 

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -1,10 +1,16 @@
 ---
-authors:
-  - Ansible Network Community (ansible-network)
-dependencies:
-  "ansible.netcommon": "*"
-license_file: LICENSE
+namespace: community
```
Comment (gundalow) [inline galaxy.yml:2]: I've renamed the repo to `https://github.com/ansible-collections/community.skydive`, thanks

File: changelogs/.plugin-cache.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+plugins:
```
Comment (gundalow) [inline changelogs/.plugin-cache.yaml]: Please remove this file

**Review by gundalow — COMMENTED** [review]
Please add `.gitignore` file https://github.com/ansible-collections/collection_template/blob/main/.gitignore

File: galaxy.yml (hunk 1 above)
Comment (lebauce) [reply galaxy.yml:2]: Thanks @gundalow !

File: changelogs/.plugin-cache.yaml (hunk 2 above)
Comment (lebauce) [reply changelogs/.plugin-cache.yaml]: Done

File: plugins/modules/skydive_capture.py [hunk 3]
```diff
@@ -86,9 +88,9 @@
     - absent
 """
 
-EXAMPLES = """
+EXAMPLES = r"""
 - name: start a new flow capture directly from gremlin query
-  skydive_capture:
+  community.skydive.skydive_capture:
```
Comment (lebauce) [inline plugins/modules/skydive_capture.py:93]: @gundalow Is it required ?

Comment (gundalow) [reply plugins/modules/skydive_capture.py:93]: Examples should use FQCN, do this change is correct.

An example of this being done in an other collection: https://github.com/ansible-collections/community.general/pull/643/files

File: changelogs/.plugin-cache.yaml (hunk 2 above)
Comment (resmo) [reply changelogs/.plugin-cache.yaml]: doesn't look like removed to me, please verify again.

Comment (lebauce) [reply changelogs/.plugin-cache.yaml]: Indeed, it was re added by mistake. Removed

**Pull request merged**